src/main/java/org/example/algorithms/searching/linear_search.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+class linear_search_array
```

Comment:
- Please rename class and file name to _LinearSearch_ to follow convention.
- Include package info at top of file as :
``` java
package org.example.algorithms.searching;
```

Reply:
ok!